### ISSUE TITLE
Once you navigate away from the welcome bin there is no way back

### ISSUE THREAD

**Issue by SteveALee** [christianalfoni/webpack-bin#92]
I simply went back in browsers history and then forward again to webpackbin

**Comment by christianalfoni** [comment]
Yeah, that is actually by design :-)

I do not want people who knows the service to always see the welcoming screen. Want them to go straight into an empty bin. But for people visiting the service the first time should be able to choose to load the welcoming bin to get some more info on how it works :-)

**Comment by SteveALee** [comment]
Yeah, that'd good. But if you add a link to it in menu we can remind ourselves. Only issue u can think of is if unsaved code

**Comment by christianalfoni** [comment]
Hm, sorry, but not quite sure what you mean @SteveALee?

How would you like it to work? :)

**Comment by SteveALee** [comment]
Just a link on help menu called something like 'open welcome bin'

**Comment by christianalfoni** [comment]
Ah, I see, yeah. That is a good idea :)
